statediff/types.go

```diff
@@ -39,27 +38,6 @@ type Payload struct {
 	BlockRlp     []byte `json:"blockRlp"`
 	ReceiptsRlp  []byte `json:"receiptsRlp"`
 	StateDiffRlp []byte `json:"stateDiff"    gencodec:"required"`
-
-	encoded []byte
-	err     error
-}
-
-func (sd *Payload) ensureEncoded() {
-	if sd.encoded == nil && sd.err == nil {
-		sd.encoded, sd.err = json.Marshal(sd)
-	}
-}
-
-// Length to implement Encoder interface for Payload
```

Comment:
We're not currently encoding the Payloads, so I removed the Encoder interface because it wasn't being used. I'm wondering if rlp encoding the Payloads is required for this first pass. 🤔